src/micro_http/src/request.rs

```diff
@@ -276,6 +267,15 @@ mod tests {
         }
     }
 
+    impl RequestLine {
+        pub fn new(method: Method, uri: &str, http_version: Version) -> Self {
+            Self {
+                method,
+                uri: Uri::new(uri),
+                http_version,
+            }
+        }
+    }
     #[test]
```

Comment:
nit: can you add an empty line between these 2?

Reply:
of course, done!